pylintrc

```diff
@@ -27,6 +27,7 @@ disable=
     too-many-boolean-expressions,
     too-many-branches,
     too-many-instance-attributes,
+    too-many-function-args,
```

Comment:
`IPAdmin_init` has too many arguments. This check didn't trigger previously when it was in `IPAdmin.__init__`